modules/ipfs-cpinner-client/src/index.ts

```diff
@@ -1,98 +1,89 @@
 import axios from 'axios'
-import { Agent, AbstractSecretBox } from 'daf-core'
-import RIFStorage, { Provider } from '@rsksmart/rif-storage'
+import { decodeJWT } from 'did-jwt'
+import authManagerFactory from './auth-manager'
+import { ACCESS_TOKEN_KEY } from './constants'
+import {
+  AuthenticationManager,
+  ClientKeyValueStorage, CreateContentPayload, CreateContentResponse,
+  DeleteTokenPayload, GetContentPayload, Config,
+  SwapContentPayload, SwapContentResponse
+} from './types'
 
-interface ICentralizedIPFSPinnerClient {
-  put: (key: string, content: string) => Promise<string>
-  get: (key: string) => Promise<{ cid: string, content: Buffer }[] | { cid: string }[]>
-  delete: (key: string, cid: string) => Promise<string>
+const ClientKeyValueStorageFactory = {
```

Comment:
I would move it to a new file and remove it from the exports. User then point it by location

Reply:
In the future we can even move it out and configure it as a pluggable, making mobile apps and such not dependent on this

Reply:
Opened https://github.com/rsksmart/rif-data-vault/issues/16

Reply:
I'm just exporting the interface I want to be implemented when sending a custom storage, I think we should keep it there